tests/foreman/ui/test_computeresource_openstack.py

```diff
@@ -0,0 +1,361 @@
+
+
+class OpenstackComputeResourceTestCase(UITestCase):
+    """Implements compute resource tests in UI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_openstack_with_name(self):
+        """Create a new openstack compute resource using different data values
+        as name
+
+        @id: 17f8e903-c2db-4580-8162-0706da6c6368
+
+        @setup: openstack hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type openstack.
+        2. Provide a valid hostname, username and password.
+        3. Provide a valid name to openstack compute resource.
+        4. Test the connection using Load Tenants and submit.
+
+        @Assert: An openstack compute resource is created successfully.
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_openstack_with_description(self):
+        """Create openstack compute resource with description.
+
+        @id: 8373b79a-bffc-409b-95fe-0bb170536c5b
+
+        @setup: openstack hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type openstack.
+        2. Provide a valid hostname, username and password.
+        3. Provide a valid description to openstack compute resource.
+        4. Test the connection using Load Tenants and submit.
+
+        @Assert: An openstack compute resource is created successfully
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_create_openstack_with_invalid_name(self):
+        """Create a new openstack compute resource with invalid names.
+        only
+
+        @id: aa0cf21d-40a2-4478-9ab5-8e18db7fb8c0
+
+        @setup: openstack hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type openstack.
+        2. Provide a valid hostname, username and password.
+        3. Provide a invalid name to openstack compute resource.
+        4. Test the connection using Load Tenants and submit.
+
+        @Assert: An openstack compute resource is not created
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_update_openstack_name(self):
+        """Update An openstack compute resource name
+
+        @id: 27b475dc-bec8-4b25-a67a-0bdba4d086c9
+
+        @setup: openstack hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type openstack.
+        2. Provide a valid hostname, username and password.
+        3. Provide a valid name to openstack compute resource.
+        4. Test the connection using Load Tenants and submit.
+        5. Update the name of the created CR with valid string.
+
+        @Assert: The openstack compute resource is updated
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
```

Comment:
imho it's a part of CR**U**D tests, but it's more of a subjective feeling. (not a blocker)